scripts/web_reports.py

```diff
@@ -31,7 +31,26 @@ def get_participation_string(participation, language):
         labels = labels,
         percentage = 0 if invited == 0 else (answered / invited * 100))
 
-def web_report_course(summary_path, stat_path, output_path, html_templates, courses):
+def create_chart_js(question, question_stats, scales, chart_id):
+    chart_data = []
+    answers = scales[question]["order"]
+    for answer in answers:
+        if answer in question_stats["counts"]:
+            count = question_stats["counts"][answer]
+        else:
+            count = 0
+        chart_data.append('{{ label: "{}", value: {} }}'.format(answer, count))
+    if len(answers) == 6:
+        colors = "['#1a9850', '#91cf60', '#d9ef8b', '#fee08b', '#fc8d59', '#d73027']"
+    elif len(answers) == 5:
+        colors = "['#d7191c', '#fdae61', '#91cf60', '#abd9e9', '#2c7bb6']"
+    else:
+        print("Warning: Chart for '{}' omitted. No colors defined for questions with {} alternatives".format(
```

Comment:
Tror egentlig vi bør putte skalatype for fargevalg i scales eller noe sånt. For er jo egentlig vesentlig hva som er bra og dårlig også. Er jo bare tilfeldig at 5 alternativer er bra på midten, og 6 er bra på starten i de spørsmålene vi har. Løsningen er sånn sett såppass hacky at jeg nesten synes det er en fordel at den er såpass restriktiv at vi sannsynligvis må gjøre et bevisst valg når vi endrer ting senere.